chapters/json/CRUD-Operations/Update-zcap-root-capability-patch.json

```
@@ -0,0 +1,28 @@
+{
+  "@context": [
+    "https://w3id.org/zcap/v1",
+    "https://w3id.org/security/data-integrity/v2",
+    "https://w3id.org/json-ld-patch/v1"
+  ],
+  "patch": [
+    {
+      "op": "add",
+      "path": "/service/4",
+      "value": {
+        "id": "#linked-domain",
+        "type": "LinkedDomains",
+        "serviceEndpoint": "https://contact-me.com"
+      }
+    }
+  ],
+  "proof": {
+    "type": "DataIntegrityProof",
+    "cryptosuite": "secp-schnorr-2024",
```

```suggestion
    "cryptosuite": "secp256k1-schnorr-jcs-2025",
```

May aswell fix this. Although will likely change again in the future